clouddriver-titus/src/main/groovy/com/netflix/spinnaker/clouddriver/titus/client/model/Task.java

```diff
@@ -40,6 +40,7 @@ public Task(com.netflix.titus.grpc.protogen.Task grpcTask) {
     startedAt = getTimestampFromStatus(grpcTask, TaskStatus.TaskState.StartInitiated);
     finishedAt = getTimestampFromStatus(grpcTask, TaskStatus.TaskState.Finished);
     containerIp = grpcTask.getTaskContextOrDefault("task.containerIp", null);
+    agentId = grpcTask.getTaskContextOrDefault("agent.instanceId", null);
```

Comment:
worth adding a comment here about what this field is used for when titus supports it we can clean it up.